docker-ci/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Parent image for zingolibs CI
+FROM parent-ci:latest
+
+# RUN git clone https://github.com/zingolabs/zingolib.git
+RUN git clone https://github.com/Oscar-Pepper/zingolib.git # test only
+
+# Copy zcash and lightwallet binaries into zingolib regtest directory
+WORKDIR zingolib
+RUN git checkout ci-update # test only
+RUN cp /usr/bin/lightwalletd /usr/bin/zcashd /usr/bin/zcash-cli ./regtest/bin/
+
+# Create CARGO_HOME directory for package caching
+# RUN mkdir ./cargo_home
+
+# ------------test only ----------------------
+RUN cargo build --release
```

Comment:
Should we add the `cross_version` feature here?

Reply:
added to cargo test entrypoint in #221